ci/qa/phpmd.xml

```diff
@@ -32,6 +32,7 @@
         <exclude name="LongVariable" />
         <exclude name="LongClassName" />
         <exclude name="ShortClassName" />
+        <exclude name="ShortMethodName" />
```

Comment:
This one can be removed IMO, unless you have a valid reason for adding it of course!

Reply:
It was not in the previous test either, therefore it can be removed